pkg/util/chunk/column.go

```diff
@@ -253,14 +304,35 @@ func (c *Column) AppendCellNTimes(src *Column, pos, times int) {
 	} else {
 		c.appendMultiSameNullBitmap(notNull, times)
 	}
-	if c.isFixed() {
+	if c.IsFixed() {
 		elemLen := len(src.elemBuf)
 		offset := pos * elemLen
+		if intest.InTest {
```

Comment:
I think we can check this once after all `AppendCellNTimes` is called in `appendProbeRowToChunkInternal`

Reply:
> I think we can check this once after all `AppendCellNTimes` is called in `appendProbeRowToChunkInternal`

 There will be no overhead in release version as this is the check only in test. Checking this once after all `AppendCellNTimes` is called in `appendProbeRowToChunkInternal` will make the codes more complex, so I think it's ok to check here.

Reply:
> But still the code need to execute the `if intest.InTest` branch

fixed now